relayer/workers/beefyrelayer/beefy-relaychain-listener.go

```diff
@@ -12,27 +9,30 @@ import (
 	"github.com/snowfork/go-substrate-rpc-client/v2/types"
 	"golang.org/x/sync/errgroup"
 
-	"github.com/snowfork/polkadot-ethereum/relayer/store"
+	"github.com/snowfork/polkadot-ethereum/relayer/chain/relaychain"
 	"github.com/snowfork/polkadot-ethereum/relayer/substrate"
+	"github.com/snowfork/polkadot-ethereum/relayer/workers/beefyrelayer/store"
 )
 
-type Listener struct {
-	config        *Config
-	conn          *Connection
-	beefyMessages chan<- store.BeefyRelayInfo
-	log           *logrus.Entry
+type BeefyRelaychainListener struct {
```

Comment:
nit: If this struct is already in a `beefyrelayer` package, can just shorten the struct name to `Listener`. Is easier to read.

Reply:
i think in this context, the name is still valuable as there are multiple listeners with different purposes/sources